setup.cfg

```diff
@@ -3,18 +3,15 @@ current_version = 1.0.0
 commit = True
 tag = True
 
-[pylint]
-ignore = CVS
-max-line-length = 120
-
 [pycodestyle]
 count = False
-ignore = E501, W503
+ignore = E501, W503, R903
 max-line-length = 120
 statistics = True
 
 [flake8]
 max-line-length = 120
+ignore = R903, F401, F403
```

Comment:
I think you can disable that for specific files, since the 403 error should be only for backends  
https://stackoverflow.com/questions/59167405/flake8-ignore-only-f401-rule-in-entire-file

and try to disable for the specific line in the  401 case

Reply:
[68411cb](https://github.com/eduNEXT/eox-nelp/pull/25/commits/68411cbcd9a3ba36e97bf3bef34ebf4fb3726449)